Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -24,16 +24,20 @@ class DashboardViewModel: ObservableObject {
         if let uid = authenticationViewModel.user?.uid {
             var eventToAdd = maintenanceEvent
             eventToAdd.userID = uid
-            
-            try? Firestore
+
+            let eventAdded = try? Firestore
```

Comment:
A `DocumentReference` is what's actually returned, and given that I'd personally change this variable name instead of calling it `eventAdded` call it `documentRef` or `documentReference`
I'll accept this PR, but just a note for the future